File: java11/src/main/java/com/chilvers/henry/acer/AcerApplication.java
package com.chilvers.henry.acer;

/**
 * Advent Calendar Enjoyment Randomizer.
 * Copyright (c) 2017 - 2022.
 *
 * @author Henry Chilvers
 */
public class AcerApplication {
    public static void main(String[] args) {
        try {
            var printer = new Printer();
            var game = new Game();

            printer.printOpeningScreen();

            game.setup(printer);

            printer.printPlayerTable(game);


            game.chooseDaysToOpen(printer);

            printer.printFinalResults(game);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: kotlin/src/main/kotlin/com/chilvers/henry/acer/AcerApplication.kt
package com.chilvers.henry.acer

/**
 * Advent Calendar Enjoyment Randomizer.
 * Copyright (c) 2017 - 2022.
 *
 * @author Henry Chilvers
 */
fun main(args: Array<String>) {
    try {
        val printer = Printer()
        val game = Game()

        printer.printOpeningScreen()

        game.setup(printer)

        printer.printPlayerTable(game)

        game.chooseDaysToOpen(printer)

        printer.printFinalResults(game)
    } catch (e: Exception) {
        e.printStackTrace()
    }
}